From 79a41ea09526d87ca7f6da95d7ad2e9f6993e784 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Sun, 3 Jan 2016 11:33:12 +0200
Subject: [PATCH 23/30] ARM: i.MX6UL: Fix ESCPI2 pads daisy chain input value

Fix ESCPI2 SCLK iand MISO daisy chain input value.
Solves the touchscree read spi data issue.

According to the i.MX6Ul Reference Manual:
1) IOMUXC_ECSPI2_SCLK_SELECT_INPUT.Daisy bit has to be 1
in order to allow the ESCPI2_SCLK work on the UART4_TX_DATA pad.
2) IOMUXC_ECSPI2_MISO_SELECT_INPUT.Daisy bit has to be 1
in order to allow the ESCPI2_MISO work on the UART5_RX_DATA pad.

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 arch/arm/boot/dts/imx6ul-pinfunc.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ul-pinfunc.h b/arch/arm/boot/dts/imx6ul-pinfunc.h
index 1bb774b9..a218e8d 100644
--- a/arch/arm/boot/dts/imx6ul-pinfunc.h
+++ b/arch/arm/boot/dts/imx6ul-pinfunc.h
@@ -270,7 +270,7 @@
 #define	MX6UL_PAD_UART4_TX_DATA__CSI_DATA12                      	0x00B4 0x0340 0x0000 3 0
 #define	MX6UL_PAD_UART4_TX_DATA__CSU_CSU_ALARM_AUT02             	0x00B4 0x0340 0x0000 4 0
 #define	MX6UL_PAD_UART4_TX_DATA__GPIO1_IO28                      	0x00B4 0x0340 0x0000 5 0
-#define	MX6UL_PAD_UART4_TX_DATA__ECSPI2_SCLK                     	0x00B4 0x0340 0x0000 8 0
+#define	MX6UL_PAD_UART4_TX_DATA__ECSPI2_SCLK                     	0x00B4 0x0340 0x0544 8 1
 #define	MX6UL_PAD_UART4_RX_DATA__UART4_DCE_RX                        	0x00B8 0x0344 0x063C 0 1
 #define	MX6UL_PAD_UART4_RX_DATA__UART4_DTE_TX                        	0x00B8 0x0344 0x0000 0 0
 #define	MX6UL_PAD_UART4_RX_DATA__ENET2_TDATA03                   	0x00B8 0x0344 0x0000 1 0
@@ -294,7 +294,7 @@
 #define	MX6UL_PAD_UART5_RX_DATA__CSI_DATA15                      	0x00C0 0x034C 0x0000 3 0
 #define	MX6UL_PAD_UART5_RX_DATA__CSU_CSU_INT_DEB                 	0x00C0 0x034C 0x0000 4 0
 #define	MX6UL_PAD_UART5_RX_DATA__GPIO1_IO31                      	0x00C0 0x034C 0x0000 5 0
-#define	MX6UL_PAD_UART5_RX_DATA__ECSPI2_MISO                     	0x00C0 0x034C 0x0000 8 0
+#define	MX6UL_PAD_UART5_RX_DATA__ECSPI2_MISO                     	0x00C0 0x034C 0x0548 8 1
 #define	MX6UL_PAD_ENET1_RX_DATA0__ENET1_RDATA00                  	0x00C4 0x0350 0x0000 0 0
 #define	MX6UL_PAD_ENET1_RX_DATA0__UART4_DCE_RTS                  	0x00C4 0x0350 0x0638 1 0
 #define	MX6UL_PAD_ENET1_RX_DATA0__UART4_DTE_CTS                  	0x00C4 0x0350 0x0000 1 0
-- 
1.9.1

